test/recipes/80-test_ssl_old.t

```diff
@@ -822,4 +822,37 @@ sub testssl {
         ok(run(test([@ssltest, "-bio_pair", "-tls1", "-ct"])));
         }
     };
+
+    subtest 'Session ticket + SNI enable tests' => sub {
+	######################################################################
```

Comment:
A few more pointers how to get this working:

You can configure ticket support directly on the client/server with the SessionTicket conf command:

https://www.openssl.org/docs/manmaster/ssl/SSL_CONF_cmd.html

So this will work out-of-the-box and you won't have to modify the test framework at all.

However you'll have to modify test/handshake_helper.c and test/generate_ssl_tests.pl to support an optional second server ctx for SNI.